src/app/dataset-view/dataset.component.ts

```diff
@@ -388,6 +282,53 @@ export class DatasetComponent implements OnInit, OnDestroy {
         this.onSelectDataset(idDataset);
     }
 
+    public getParentMethod(): { [key: string]: (...args: any[]) => any } {
```

Comment:
This should become a delegate interface instead of a dictionary

Reply:
fixed